src/ui/Icon/Icon.tsx

```diff
@@ -13,7 +13,7 @@ const iconComponentCollection = {
 type IconCollection = typeof iconComponentCollection
 type Variant = keyof IconCollection
 
-type OutlineIconCollection = typeof svgOutline
+export type OutlineIconCollection = typeof svgOutline
```

Comment:
Exported this type so I can declare a custom type in `TextInput.tsx` based on the keys of this collection. `TextInput` renders an Icon with an `outline` variant, which type checks against keys only in this collection.